views/course.handlebars

```diff
@@ -5,20 +5,47 @@
   <link rel="stylesheet" href="../css/bulma.css">
 </head>
 <body>
-  {{> hero}}
-  <section class="hero is-fullheight is-default is-bold">
-    <div class="hero-body">
-      <div class="container has-text-centered">
-        <div class="columns is-vcentered">
-          {{> courseImage}}
-          <div class="column is-6 is-offset-1">
-            {{> courseDescription}}
-            <br>
+  <section class="hero is-primary">
+    {{> header}}
+    <div class="hero-body columns max-height">
+      <div class="column is-half ">
+        <div class="content full-hight">
+        <p>
+        <h1 class="title is-2 ">
```

Comment:
Porque o uso da classe is-2 aqui?

Reply:
is-2 é necessário para definir o tamanho do título
segue o link que utilizamos para definir os tamanhos:
https://bulma.io/documentation/elements/title/